### DISCUSSION TITLE
It works but it sucks. no?

### DISCUSSION THREAD

**Discussion by rustyvanilla** [codetheweb/muse#1154] [General]
compared to Jagrosh's (when it was working) this thing is ... wait ....... "**thinking**" ..... so fucking slow, and half the time it replies with "application did not respond"

It's not the dev's fault, Node.js bloat and the billion packages it installs are probably sucking all the resources of my server

I'm thankful it works, and it's unfortunately the best we have right now

**Comment by DaanSelen** [comment]
Jagrosh's bot can't play youtube no longer out of the box. Which in itself would make this better. I came from Jagrosh's bot, trying this out now.

**Reply by rustyvanilla** [reply]
My post was more of a response to the dev's description "🎧 a self-hosted midwestern Discord music bot that doesn't suck"

Try ErdbeerbaerLP fork of Jagrosh's - it worked for me, and it's a 100 times faster and more efficient than this.

**Reply by DaanSelen** [reply]
> ErdbeerbaerLP

Does that support Spotify?
